lib/components/molecules/QuoteMedia/QuoteMedia.js

```diff
@@ -6,36 +6,33 @@
  */
 import React from 'react';
 import styled from 'styled-components';
+import classnames from 'classnames';
 
 import type { Node } from 'react';
 import styles from './QuoteMedia.style';
 import type { Props } from './types';
 
 import Heading from '../../atoms/Heading/Heading';
 
-const QuoteMedia = ({
-  className,
-  title,
-  quoteContent,
-  HeadingType,
-  variant,
-  children,
-}: Props): Node => (
-  <div className={className}>
-    <Heading HeadingType={HeadingType} className="title">
-      {title}
-    </Heading>
-    <div className={`${className} ${variant}`}>
+const QuoteMedia = ({ className, title, quoteContent, HeadingType, children }: Props): Node => (
+  <div className={classnames(className)}>
+    {title && (
+      <Heading HeadingType={HeadingType} className="title">
+        {title}
+      </Heading>
+    )}
```

Comment:
title should be a checkpoint  - if title is there then only Heading component render.